src/interfaces/posts.ts

```diff
@@ -72,43 +69,46 @@ export interface PostComponentProps {
     /** The URL to the post. */
     url: string
 
-    className?: string
     headerClassName?: string
     titleClassName?: string
     titleLinkClassName?: string
-    tag?: 'li' | 'div'
     renderTitleAsLink?: boolean
+    className?: string
+    tag?: 'li' | 'div' | 'article'
+    contentClassName?: string
 }
 
 export interface PostIndexComponentProps {
-    posts: PostIndexItem[]
-    allPosts: PostIndexItem[]
+    posts: PostIndexItemProps[]
+    allPosts: PostIndexItemProps[]
 }
 
-export interface PostIndexItem {
+export interface PostIndexItemProps {
     frontmatter: FrontMatter
     excerpt: string | MDXRemoteSerializeResult
     slugPath: string
     className?: string
     headerClassName?: string
     titleClassName?: string
     titleLinkClassName?: string
-    tag?: 'li' | 'div'
+    tag?: 'li' | 'div' | 'article'
     renderTitleAsLink?: boolean
+    blogType: BlogType
```

Comment:
In a similar vein to naming convention, this is nit-picky, but I'd prefer we call these variables `postType` or `articleType` to be distinctive between our blogs,  podcasts, releases content. And if `components/Blog` directory was ▶️ `components/Article` or `components/Post`. What do you think? Obviously this is non-blocking and the PR is merged

Reply:
Yes! Love that! Let's go with `postType`. It makes a lot more sense since not everything is a blog. I think this is pretty standard in the CMS world when it comes to posts (ie: WordPress).